client-device-connection-infinispan/src/main/java/org/eclipse/hono/deviceconnection/infinispan/client/CommonCacheOptions.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.deviceconnection.infinispan.client;
+
+import org.eclipse.hono.util.CommandRouterConstants;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.ConfigMapping.NamingStrategy;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Common options for configuring a cache.
+ *
+ */
+@ConfigMapping(prefix = "hono.commandRouter.cache.common", namingStrategy = NamingStrategy.VERBATIM)
```

Comment:
Having the `commandRouter` reference here doesn't fit the nature of the module this class is in very well. (Although I'm aware it works because the only place where the module is used in a quarkus component is the command-router.)

But, would it also work if this annotation gets moved to a sub-interface, defined in the command router module?

Reply:
I guess so, let me check it out ...

Reply:
But we can also simply use a more generic (and unused) prefix here like `hono.infinispan` and then only override the prefix in the command router module. We then do not need to define an empty interface just for hosting the annotation ...

Reply:
Sounds good.

Reply:
I have pushed corresponding changes

Reply:
Does this work for you?

Reply:
Yes.